src/test/java/com/onaio/steps/adapter/HouseholdAdapterTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2016. World Health Organization
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.onaio.steps.adapter;
+
+import android.view.View;
+import android.widget.TextView;
+
+import com.onaio.steps.R;
+import com.onaio.steps.activities.HouseholdListActivity;
+import com.onaio.steps.adapters.HouseholdAdapter;
+import com.onaio.steps.model.Household;
+import com.onaio.steps.model.InterviewStatus;
+
+import junit.framework.Assert;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Config(emulateSdk = 16, manifest = "src/main/AndroidManifest.xml")
+@RunWith(RobolectricTestRunner.class)
+public class HouseholdAdapterTest {
+
+    private HouseholdAdapter adapter;
+    private List<Household> householdList;
+
+    @Before
+    public void setUp() {
+        householdList = new ArrayList<>();
+        householdList.add(getHousehold());
+
+        HouseholdListActivity householdListActivity = Mockito.spy(Robolectric.buildActivity(HouseholdListActivity.class).create().resume().get());
+        adapter = new HouseholdAdapter(householdListActivity, householdList);
+    }
+
+    @Test
+    public void testGetCountShouldReturnOne() {
+        Assert.assertEquals(1, householdList.size());
+    }
+
+    @Test
+    public void testGetItemShouldVerifyTheData() {
+        Household household = (Household) adapter.getItem(0);
+        verify(household);
```

Comment:
Nice work! :clap:  Try and make it easier by renaming the method from `verify` or moving the code back here. I initially thought that this was `Mockito.verify` with a static import